test/network/BancorNetwork.ts

```diff
@@ -3853,6 +3958,10 @@ describe('BancorNetwork', () => {
             });
 
             it('should cancel a pending withdrawal request', async () => {
+                const withdrawalRequest = await pendingWithdrawals.withdrawalRequest(id);
+                const val = await network.connect(provider).callStatic.cancelWithdrawal(id);
```

Comment:
Similar to the above, it'd be clearer to give it more specific name

Reply:
Fixed (in all other places as well)